.rubocop.yml

```diff
@@ -45,6 +45,8 @@ Metrics/ParameterLists:
   CountKeywordArgs: false
 Metrics/LineLength:
   Max: 100
+  Exclude:
+    - lib/apollo-federation/tracing/proto/apollo_pb.rb
```

Comment:
Do you actually need to exclude this file? I think you could probably clean up the line lengths in the file.

Reply:
@bagelbits This file was automatically generated by protoc, so I didn't want to edit it!